spatialprofilingtoolbox/apiserver/app/validation.py

```diff
@@ -13,24 +13,29 @@ def abbreviate_string(string: str) -> str:
         abbreviation = abbreviation + '...'
     return abbreviation
 
-async def valid_study_name(study: str=Query(min_length=3)) -> str:
+
+async def valid_study_name(study: str = Query(min_length=3)) -> str:
```

Comment:
Ah, I missed the memo about this case. I thought that the default expressions for keyword arguments expressions were supposed to NOT pad the `=` with spaces according to some style mandate. It turns out that this is true normally but not when the argument is also type hinted (PEP8).
There are many places that I have been using the non-recommended style here.